### DISCUSSION TITLE
¡Feliz Día Internacional de la Mujer!

### DISCUSSION THREAD

**Discussion by KasturiSShah** [PatronesHermosos/design-thinking-1ra-generacion#20] [General]
Hola, buenos días,

 🎆 💪 ¡Feliz Día Internacional de la Mujer! 🎆 💪 

Las invitamos a compartir pláticas inspiradoras / eventos interesantes / libros favoritos para celebrar el día de hoy, por medio de este hilo en el foro. De esa manera, podemos disfrutar de estos recursos juntas.

¡Que tengan lindo día! 😄

**Comment by jennyperez28** [comment]
Qué detalle Kasturi!
Un feliz día para todas, recordando que cada una haga valer hoy y siempre los derechos por los que otras valientes, años atras, lucharon por nosotras y las que vienen. 
Un abrazo.

**Comment by sowisla** [comment]
Por el Día Internacional de la Mujer, me gustaría compartir que uno de mis libros favoritos de ciencia ficción es _Frankenstein_ o _el moderno Prometeo_ de Mary Shelley. Les dejo un video de Ted Ed para que se animen a leerlo. (: https://youtu.be/PDgu25Dsv34
